DockerfileWindows

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Sonobuoy contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM BASEIMAGE
+MAINTAINER John Schnake "jschnake@vmware.com"
+
+ADD build/windows/amd64/sonobuoy.exe /sonobuoy.exe
```

Comment:
Should `build/windows/amd64/sonobuoy.exe` be `BINARY` as that's what the Makefile is expecting to replace.

Reply:
👍 thanks; started out with hardcoding and didnt revert that part on the next iteration I guess.